README.md

```diff
@@ -22,6 +22,9 @@ that lives behind an OVS bridge and has its own unique interfaces.
 #### Step 1: Clone the project:
 
 ```
+$ export GOPATH=`pwd`
```

Comment:
This is minor and feel free to ignore. Otherwise change LGTM.

Most of the users with go environment will have $GOPATH set and if not we can let standard go utilities to test and fail if needed than suggesting that user set their GOPATH to `pwd`.

I think we can have instructions as simple as:

```
go get -d https://github.com/contiv/netplugin
cd $GOPATH/src/github.com/contiv/netplugin
make build demo ssh
```